### PR TITLE
Merge fixes

### PR DESCRIPTION
**Pull request by ostromart**
A bunch of problems fixed due to heavy merging activity:
 - update code with proto naming changes (e.g. defaultNamespacePrefix changed)
 - package source selection has changed (file:/// no longer required for local path)

The current code still has one breakage: it does not correctly set ingress / egress gateway enablement and namespaces. This requires some path selection work to be fixed.

### CODE REVIEW

File: pkg/helm/helm.go [hunk 1]
```diff
@@ -118,6 +123,10 @@ func renderChart(namespace, baseValues, overlayValues string, chrt *chart.Chart)
 
 	var sb strings.Builder
 	for _, f := range files {
+		// add yaml separater if the rendered file doesn't have one at the end
```
Comment (richardwxn) [inline pkg/helm/helm.go]: nit: separator

**Comment by richardwxn** [comment]
Actually it would be great if you can give details about what are the problems that this PR tries to fix because we might be missing contexts for some of them

File: cmd/mesh/manifest.go [hunk 2]
```diff
@@ -28,7 +28,8 @@ import (
 	"istio.io/operator/pkg/translate"
 	"istio.io/operator/pkg/util"
 	"istio.io/operator/pkg/validate"
-	"istio.io/operator/pkg/version"
+
+	binversion "istio.io/operator/version"
```
Comment (ayj) [inline cmd/mesh/manifest.go]: Why is this alias needed? Also, there's an extra space between `validate` and `version` packages.

Comment (ostromart) [reply cmd/mesh/manifest.go]: I added this for consistency - we now have two version packages: a library under pkg/version and the compiled in version in version/version (which we could rename I suppose).

File: pkg/helm/helm.go (hunk 1 above)
Comment (ostromart) [reply pkg/helm/helm.go]: Done.

**Comment by ostromart** [comment]
@richardwxn I updated the PR description to give some more context.

**Pull request merged**